js/libs/form.js

```diff
@@ -453,6 +466,8 @@ Fliplet().then(function () {
             action: 'submit'
           });
 
+          isResetAction = false;
```

Comment:
Will it make sense to set `isResetAction` to `false` at the end of `resetForm()` instead?

Reply:
Hi @tonytlwu 

We can't do isResetAction = false in resetForm, due to parallel method execution, please refer [this](https://weboo.atlassian.net/browse/XOR-718?focusedCommentId=40251) comment for your reference

cc @yogeshmsharma-xoriant

Reply:
@NikitaJadhavXoriant Then perhaps `isResetAction` should be set to `false` in the `submit()` function only right before calling `resetForm()`?